homeassistant/components/zwave/__init__.py

```diff
@@ -678,12 +682,24 @@ def network_value_changed(self, value):
 
     def value_changed(self, value):
         """Called when a value for this entity's node has changed."""
+        self.update_attributes()
         self.update_properties()
         self.schedule_update_ha_state()
 
+    def update_attributes(self):
+        """Update the node attributes. May only be used inside callback."""
+        self.node_id = self._value.node.node_id
+        self.location = self._value.node.location
+        self.battery_level = self._value.node.get_battery_level()
+        self.wakeup_interval = _get_wakeup(self._value.node)
```

Comment:
Since this is no longer called from `device_state_attributes` inline the function here without the reties.